.github/workflows/ci.yml

```diff
@@ -94,13 +92,11 @@ jobs:
         with:
           python-version: 3.8
       - name: Install bazel
-        run: ./ci/run_ci.sh install_bazel
-      - name: Install python
-        run: ./ci/run_ci.sh install_python
-      - name: Install pyfory
-        run: ./ci/run_ci.sh install_pyfory
+        run: python ./ci/run_ci.py cpp --install-deps-only
+      - name: Install python dependencies
+        run: pip install pyarrow==15.0.0 Cython wheel pytest setuptools -U
```

Comment:
If we move this to run_ci.py, we could comute arrow version from python version. i.e. using pyarrow 15 when running py3.8, otehrwise using pyarrrow 18.0

Reply:
I agree. I try to minimize GHA usage.